pynicotine/plugins/core_commands/__init__.py

```diff
@@ -537,6 +549,35 @@ def list_shares_command(self, args, **_unused):
 
         self.output("\n" + f"{num_listed} shares listed ({num_total} configured)")
 
+    def share_command(self, args, **_unused):
+
+        args_split = args.split(maxsplit=1)
+        group_name, folder_path = args_split[0], args_split[1].strip(' "')
+
+        self.core.shares.remove_share(folder_path)
+        virtual_name = self.core.shares.add_share(group_name, folder_path)
+
+        if not virtual_name:
+            self.output(_("Cannot share inaccessible folder \"%s\"") % folder_path)
```

Comment:
Error is incorrectly printed, yet the folder can be accessed but it is already shared.